### ISSUE TITLE
Fixes needed for the final Version

### ISSUE THREAD

**Issue by DaCHRIS** [pirl/Oystr#1]
1 - The Windows Installation doesn't download a working node 
2 - Currency List (while changing the shown currency) hasn't USD and EUR in it, instead of this is has "PIRL" in it as currency several Times. 
3 - Import Account Window says something about the Pirl Pre Sale in 2014, this must be wiped out. 
4 - New Transaction Screen and New Contract Screen have each several times "Ether" instead of Pirl into it 
5 - Windows Wrapper restarts the Wallet alon after some times without a reason

**Comment by MatWaller** [comment]
Point 5 can you run with command line and give me the console output.

Would be interesting to see if there are any errors when it restarts

**Comment by MSFTserver** [comment]
@wallermadev there is none, the app closes and re-opens and connects but nothing in the console

**Comment by trevhacker187** [comment]
i cant get pirl gui to sync with node

**Comment by sysdynamix** [comment]
After starting the windows executable, the machine can no longer use a web browser.  Seems like port 80/443 is being proxied or something.  Windows diagnostics reports that the site is reachable (www.google.com) but cannot accept a connection.  I have tried it on 2 different machines (each fresh windows installs). Prior to launching the wallet - the machine can access any website. Any ideas?

**Comment by MaxFreedom123** [comment]
How does one BACKUP the Oystr wallet on Windows?

The Backup feature in the menu does NOT appear to work.

**Comment by trevhacker187** [comment]
it  dont work i tryd that

**Comment by jernest93** [comment]
Can anyone help me out here?
First I have tried to create a wallet contract but it never was successful.
Second I already mined like 35 PIRL credited into my main wallet, I sent it to my exchange wallet and it has been more than an hour and I got 0 confirmation of 12. I was using PirlOystrLite Version 0.0.1 but doing some research I find out that it was a new version nautilus, so I decided to backup my account and imported into the new version and now I got 0 coins and no transaction  
![image](https://user-images.githubusercontent.com/35437610/34923135-fab0b7a8-f966-11e7-9ced-3f9697a1ed43.png)
![image](https://user-images.githubusercontent.com/35437610/34923382-5727204c-f969-11e7-9ba0-2ff3e6b4212e.png)
![image](https://user-images.githubusercontent.com/35437610/34923404-98e17a96-f969-11e7-9842-19ea665479ef.png)
